File: src/main/java/no/nav/melosys/eessi/models/sed/nav/Bruker.java

package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import lombok.Data;

import java.util.List;

@SuppressWarnings("unused")
@JsonInclude(Include.NON_NULL)
@Data
public class Bruker {

    private List<Adresse> adresse;

    private Far far;

    private Mor mor;

    private Person person;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Bruker.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Bruker(
    var adresse: List<Adresse>? = null,
    var far: Far? = null,
    var mor: Mor? = null,
    var person: Person? = null
)
